model/Licensing/Classes/OrLaterOperator.md

```diff
@@ -23,7 +25,7 @@ data will need to determine for themselves what meaning to attribute to a
 ## Metadata
 
 - name: OrLaterOperator
-- SubclassOf: AnyLicenseInfo
+- SubclassOf: LicenseExpression
```

Comment:
In the Tooling Mini Summit one use case for the licensing profile was the following license expression "GPL-3.0-or-later WITH AdditionRef-autoconf-gpl3". This implies that we need to be able to use the `OrLaterOperator` as `license` in the `WithOperator`. One option would be to make the `OrLaterOperator` a subclass of `License` but as this would then allow some kind of "nested" `OrLaterOperator` which is not valid on the other hand, we probably need to rethink the structure here. As this is also a bug with the current model, I'll open an issue for this.

Reply:
Please note that this specific example is not using the OrLater operator; it's a single license named `GPL-3.0-or-later`.

I agree that we should clarify whether OrLater can be used with With.

Reply:
Ah sorry, bad example, but "Apache-1.0+ WITH AdditionRef-autoconf-gpl3" would be an example for this use case, right? I will update the example in the issue.

Reply:
legacy but still a valid SPDX expression that can be used as example: `GPL-3.0+ WITH AdditionRef-autoconf-gpl3`